<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dicomWindow</class>
 <widget class="QDialog" name="dicomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dicom Viewer</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="ipView">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>320</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Inphase View</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="opView">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>320</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Outphase View</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>165</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="ButtonLayout">
    <item row="4" column="1">
     <widget class="QPushButton" name="zmOPBtn">
      <property name="text">
       <string>Zoom OP View</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="laodBtn">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="prevBtn">
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="uName"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="nextBtn">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="zmIPBtn">
      <property name="text">
       <string>Zoom IP View</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="roiBtn">
      <property name="text">
       <string>ROI Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>0</y>
     <width>191</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="DataLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="headLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Meta Data</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="stnNameLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Station Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="pIDDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="mnfLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Manufacturer</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="pxlSpaceLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pixel Spacing</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="pNameLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Patient Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="sosClassDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="instNameLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Inistitution Name</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="instNameDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="mnfDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="stdyDiscLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Study Discription</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="modalityDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="stnNameDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="pIDLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Patient ID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="pNameDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="stdyDiscDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="stdyDateLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Study Date</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="pxlSpaceDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="slicingLocLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Slice Location</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="stdyDateDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="slicingLocDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="imgSizLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Image Size</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="sosClassLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SOP Class</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="modalityLbl">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Modality</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="imgSizDat">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
       </font>
      </property>
      <property name="text">
       <string>Nil</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="brightness">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Brightness</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="contrast">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Contrast</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="SliderLayout">
    <item>
     <widget class="QSlider" name="brightSlid">
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>128</number>
      </property>
      <property name="sliderPosition">
       <number>128</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="contrastSlid">
      <property name="maximum">
       <number>128</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>64</number>
      </property>
      <property name="sliderPosition">
       <number>64</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
